changelog/4.2.2.md

```diff
@@ -0,0 +1,43 @@
+# Changelog for version 4.2.2
+
+### Released Sept 10 2021
+
+## Frontend Changes
+* Various UI/UX cleanup.
+* Improved contrast of highlighting in Query Studio.
+* Improved table renderer on Query Studio.
+* Fixed issue where some investigative dashboards were not cleaning up queries quickly when completed.
+* Improved query cleanup logic on Query Studio.
+* Fixed issue where System Hardware stats stop updating when an indexer is down.
+* Fixed issue where history from Query Studio was logging expanded query and stripping formatting.
+* Improved time locking and zoom when user has custom timeframes defined.
+* Improved kit refresh after kit install.
+* Fixed issue where some point2point renderer images were not properly displaying lines.
+* Improved default info handling when adding new users.
+* Improved overview stats so that bucketing lines up on zoom.
+* Improved date detection when zooming on overview chart.
+
+## Backend Changes
+* Fixed issue where some large queries could cause windowing errors in overview chart.
+* Improved webserver compression on HTTP requests.
+* Reduced memory usage when distributing very large resources between webserver and indexers.
+* Fixed issue where the `limit` module was not generating accurate stats.
+* Added version handshake between datastore and webservers.
```

Comment:
This probably needs to be called out in its own section like we did in the 4.1.0 release notes. Old datastores don't have the ability to throw a meaningful error so they'll honk with stuff like "EOF", "IO timeout" etc...